rollup.config.js

```diff
@@ -1,25 +1,31 @@
 import babel from 'rollup-plugin-babel';
 import filesize from 'rollup-plugin-filesize';
 import uglify from 'rollup-plugin-uglify';
+import { minify } from 'uglify-es';
+import { name } from './package.json';
 
 const isProduction = process.env.NODE_ENV === 'production';
 
 const destBase = 'dist/normalizr';
 const destExtension = `${isProduction ? '.min' : ''}.js`;
 
 export default {
-  entry: 'src/index.js',
-  moduleName: 'normalizr',
-  targets: [
-    { dest: `${destBase}${destExtension}`, format: 'cjs' },
-    { dest: `${destBase}.umd${destExtension}`, format: 'umd' },
-    { dest: `${destBase}.amd${destExtension}`, format: 'amd' },
-    { dest: `${destBase}.browser${destExtension}`, format: 'iife' }
+  input: 'src/index.js',
+  output: [
+    { file: `${destBase}${destExtension}`, format: 'cjs' },
+    { file: `${destBase}.es${destExtension}`, format: 'es' },
+    { file: `${destBase}.umd${destExtension}`, format: 'umd', name },
+    { file: `${destBase}.amd${destExtension}`, format: 'amd', name },
+    { file: `${destBase}.browser${destExtension}`, format: 'iife', name }
   ],
   plugins: [
-    babel({ babelrc: false, presets: [ 'es2015-rollup', 'stage-1' ] }),
-    isProduction && uglify(),
+    babel({
```

Comment:
Something that annoys me about rollup-plugin-babel... is there any way to inherit from the base config so that we're not duplicating?